docker/php.Dockerfile

```diff
@@ -46,16 +46,10 @@ RUN docker-php-source extract && \
     mkdir /usr/src/php/ext/amqp && \
     tar --extract --file amqp.tar.gz --directory /usr/src/php/ext/amqp --strip 1 && \
     docker-php-ext-install -j$(nproc) amqp && \
-    docker-php-source delete
-
-RUN docker-php-source extract && \
```

Comment:
What is the reason for removing these commands?

Reply:
If you look at the `docker-php-source` script, it will check if the source already exists before extracting it again, so there was no need to do a sequence of extract/delete/extract :-)